spec/traco_spec.rb

```diff
@@ -113,14 +113,20 @@
     expect(Post.locale_columns(:title)).not_to include(:title_ru)
   end
 
-it "supports multiple attributes" do
-  Post.translates :body
+  it "supports multiple attributes" do
+    Post.translates :body
```

Comment:
Is this hunk mistyped whitespace?

Reply:
These two lines were not indented correctly, so I thought I can fix it.
I can remove the change though if you want.